### PR TITLE
feat(route): add new LinkedIn route params

### PR DESCRIPTION
**Pull request by zhoukuncheng**
## Involved Issue / 该 PR 相关 Issue

Close #

## Example for the Proposed Route(s) / 路由地址示例

```routes
/linkedin/jobs/F/4
/linkedin/jobs/F/4/software%20engineer
/linkedin/jobs/F/4/software%20engineer/f_WT=2&geoId=91000003&f_TPR=r2592000
```

## New RSS Route Checklist / 新 RSS 路由检查表
  
- [ ] New Route / 新的路由
  - [x] Follows [Script Standard](https://docs.rsshub.app/joinus/advanced/script-standard) / 跟随 [路由规范](https://docs.rsshub.app/zh/joinus/advanced/script-standard)
- [ ] Anti-bot or rate limit / 反爬/频率限制
  - [ ] If yes, do your code reflect this sign? / 如果有, 是否有对应的措施?
- [ ] [Date and time](https://docs.rsshub.app/joinus/advanced/pub-date) / [日期和时间](https://docs.rsshub.app/zh/joinus/advanced/pub-date)
  - [ ] Parsed / 可以解析
  - [ ] Correct time zone / 时区正确
- [ ] New package added / 添加了新的包
- [ ] `Puppeteer`

## Note / 说明

增加新的参数 `work_types, geo_id, time_posted_range`，使LinkedIn订阅更可用  
参数含义参考

https://github.com/marcothedeveloper123/jobseeker  
https://www.kaggle.com/code/hachemsfar/linkedin-jobs-scraper-job-description
```
Past_month="f_TPR=r2592000"
Past_week="f_TPR=r604800"
Past_24_hours="f_TPR=r86400"

onsite="f_WT=1"
remote="f_WT=2"
hybrid="f_WT=3"
```

### CODE REVIEW

**Comment by zhoukuncheng** [comment]
/test

```
/linkedin/jobs/F/4/software%20engineer/2/91000012/r604800
```

**Comment by zhoukuncheng** [comment]
/test

```routes
/linkedin/jobs/F/4/software%20engineer
/linkedin/jobs/F/4/software%20engineer/2/91000012/r604800
```

File: lib/routes/linkedin/jobs.ts [hunk 1]
```diff
@@ -1,16 +1,36 @@
...
+    TIME_POSTED_RANGE_QUERY_KEY,
+} from './utils';
 
 const BASE_URL = 'https://www.linkedin.com/';
 const JOB_SEARCH_PATH = '/jobs-guest/jobs/api/seeMoreJobPostings/search';
 
 export const route: Route = {
-    path: '/jobs/:job_types/:exp_levels/:keywords?',
-    categories: ['other'],
+    path: '/jobs/:job_types/:exp_levels/:keywords?/:work_types?/:geo_id?/:time_posted_range?',
```
Comment (TonyRL) [inline lib/routes/linkedin/jobs.ts:23]: This adds way too many optional parameters. Please use a single optional parameter which accepts them [in URL search string style](https://docs.rsshub.app/routes/social-media#twitter) instead.

File: lib/routes/linkedin/jobs.ts [hunk 2]
```diff
@@ -19,8 +39,13 @@ export const route: Route = {
...
         supportScihub: false,
     },
+    radar: [
+        {
+            source: ['www.linkedin.com/jobs/search'],
+        },
+    ],
     name: 'Jobs',
-    maintainers: [],
+    maintainers: ['zhoukuncheng'],
```
Comment (TonyRL) [inline lib/routes/linkedin/jobs.ts:48]:
```suggestion
    maintainers: ['BrandNewLifeJackie26', 'zhoukuncheng'],
```
#11656

File: lib/routes/linkedin/jobs.ts (hunk 1 above)
Comment (zhoukuncheng) [reply lib/routes/linkedin/jobs.ts]: modified to `path: '/jobs/:job_types/:exp_levels/:keywords?/:routeParams?',`

**Pull request merged**